src/UI/molecules/PerformanceItem.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { memo } from "react";
+import styled, { css, FlattenSimpleInterpolation } from "styled-components";
+
+import FlexContainer from "<layout>/FlexContainer";
+import FlexItem from "<layout>/FlexItem";
+import SpacingContainer from "<layout>/SpacingContainer";
+
+import Text from "<atoms>/Text";
+
+import {
+  PerformanceItemBarchartProps,
+  PerformanceItemProps
+} from "<molecules>/__typings__/PerformanceItem.d.ts";
+
+function PerformanceItem({ animationDelay, title }: PerformanceItemProps): JSX.Element {
+  return (
+    <PerformanceItem.Container
+      data-testid="PerformanceItem"
+    >
+      <SpacingContainer
+        paddingBottom="spacing2"
+        paddingTop="spacing2"
+        width="100%"
+      >
+        <FlexContainer
+          flexFlow="row nowrap"
+        >
+          <FlexItem
+            flex="0 1 40%"
+          >
+            <Text
+              color="blue100"
+              ellipsis
+              fontSize="font8"
+              paddingRight="spacing4"
+              textAlign="right"
+              textTransform="uppercase"
+            >
+              {title}
```

Comment:
```suggestion
              {label}
```
`label` would be a better option in this case